### PR TITLE
[2646] We need an internal facing index of created projects

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -23,16 +24,49 @@ module.exports = function routesCtor( app, User, filter, sanitizer, stores, util
...
+    function( req, res ) {
+      Project.findRecentlyUpdated( { limit: req.params.limit }, function( err, projects ) {
+        if ( err ) {
+          res.json( { error: err }, 500 );
+        }
+        res.json( { error: 'okay', results: projects } );
+      });
+  });
+
+  app.get( '/api/projects/remixes/:id?',
```
Comment (jbuck) [inline routes/index.js]: /ap/project/remixes, since you're only looking at a single users remixes.

**Comment by humphd** [comment]
Still waiting on Travis for this...

File: routes/index.js (hunk 1 above)
Comment (jbuck) [inline routes/index.js]: Actually, better yet, `/api/project/:id/remixes` right? You'll probably need to move this definition above `/api/project/:id` though

File: server.js [hunk 2]
```diff
@@ -119,12 +119,17 @@ app.post( '/api/publish/:id',
   var email = req.session.email,
       id = parseInt( req.params.id, 10 );
 
+  if ( !email ) {
```
Comment (jbuck) [inline server.js]: You don't need to add this code; the `filter.isLoggedIn` middleware for this route will prevent the execution of this code.

File: routes/index.js [hunk 3]
```diff
@@ -23,16 +24,65 @@ module.exports = function routesCtor( app, User, filter, sanitizer, stores, util
...
+        author: result.author,
+        remixedFrom: result.remixedFrom,
+        createdAt: result.createdAt,
+        updatedAt: result.updatedAt
+      });
+    });
+    return pruned;
+  }
+
+  app.get( '/api/projects/recentlyCreated/:limit?',
```
Comment (jbuck) [inline routes/index.js:48]: `/api/project/:limit/recentlyCreated`

Comment (jbuck) [inline routes/index.js:48]: Hmm, actually....

**Pull request merged**